### ISSUE TITLE
[STILL WIP]

### ISSUE THREAD

**Issue by ghost** [MHIGists/RPGLike#4]
please close this issue when the plugin is ready for production servers.

**Comment by Johnsonk79** [comment]
You sound pretty mean ngl...

**Comment by ghost** [comment]
no lol i just wanted a notification so that i know i can use it, nothing else.

**Comment by Johnsonk79** [comment]
A better way to say it would be:
Hello! Can you please close this issue when the plugin is ready for production servers? Thanks!

**Comment by ghost** [comment]
all roads lead to roma 😉

**Comment by llPushkarSinghll** [comment]
Is The Plugin Still In WIP ?

**Comment by MHIGists** [comment]
Yes it is. The development was freezed but last month has been heavy development. There should be a release after a week or two. Until then you can try the features so far with build #135 or #136 those were the last stable development builds 

> Is The Plugin Still In WIP ?

**Comment by pushkar-pro** [comment]
Is it ready?

**Comment by PushkarOP** [comment]
Is it ready?
